app/src/main/res/layout/subscription_item.xml

```diff
@@ -1,55 +1,97 @@
-        app:backgroundColor="?attr/colorSecondary"
-        app:corner="rightTop"
-        app:primaryText="Test"
-        app:primaryTextColor="?attr/colorOnSecondary"
-        app:primaryTextSize="12sp" />
+        <LinearLayout
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:orientation="vertical">
+
+            <androidx.cardview.widget.CardView
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:clickable="false"
+                app:cardCornerRadius="8dp"
+                app:cardElevation="0dp">
+
+                <RelativeLayout
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content">
+
+                    <de.danoeh.antennapod.ui.common.SquareImageView
+                        android:id="@+id/imgvCover"
+                        android:layout_width="match_parent"
+                        android:layout_height="match_parent"
+                        android:scaleType="fitCenter"
+                        android:outlineProvider="background"
+                        squareImageView:direction="width"
+                        tools:src="@tools:sample/avatars" />
+
+                    <TextView
+                        android:id="@+id/countViewPill"
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:text="3"
+                        android:layout_alignParentEnd="true"
+                        android:textSize="12sp"
+                        style="@style/TextPill"
+                        app:primaryText="Test"
+                        app:primaryTextColor="?attr/colorOnSecondary"
+                        app:primaryTextSize="12sp" />
+
+                    <TextView
+                        android:id="@+id/fallbackTitle"
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:layout_centerInParent="true"
+                        android:layout_centerHorizontal="true"
+                        android:ellipsize="end"
+                        android:gravity="center"
+                        android:padding="6dp"
+                        android:textColor="?android:attr/textColorPrimary"
+                        tools:text="@string/app_name" />
+
+                </RelativeLayout>
+
+            </androidx.cardview.widget.CardView>
+
+            <TextView
+                android:id="@+id/txtvTitle"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:ellipsize="end"
+                android:gravity="start"
+                android:text="final text"
+                android:textColor="?android:attr/textColorPrimary"
+                android:lines="2"
```

Comment:
I guess it would be complex; would it be possible to set the row height to the maximum space needed on a given row (if nr of cards/row is 2-3, because if more cards the number of lines would get too big given the limited width)? E.g. given two cards on a row, which each have short title, there now is whitespace (see screenshot in other comment).

[Although this point may fall away if the font size gets increased as per the other comment.)

Reply:
Wouldn't this change make row height inconsistent between rows?
(Also, since we will increase font size as suggested on previous comment, should be fine, let me know on this)

Reply:
Yes, let's see if still needed after increasing the podcast title font size.